### DISCUSSION TITLE
Sometimes the select directory step in template library is blocked

### DISCUSSION THREAD

**Discussion by 1800joe** [noop-inc/documentation#82] [Bugs] [answered]
Eventually the loading indicator goes away and selecting the directory is enabled, but sometimes it takes ~30s to happen.

<img width="756" alt="Screenshot 2024-01-23 at 9 19 35 AM" src="https://github.com/noop-inc/documentation/assets/3111670/0f56ce51-5d64-4ceb-a9ce-4f1768fe0a52">

**Comment by derekwolpert** [comment, marked as answer]
There is a tweak readied for the next release of Noop Desktop which should significantly reduce the loading here. Please reply if you are still experiencing this issue with releases later than `v0.4.1`.
